0019-glossary/0019-glossary.md

```diff
@@ -0,0 +1,272 @@
+A connector receives a local transfer on one ledger in exchange for making another local transfer on a different ledger. A single interledger payment may include multiple connectors and may traverse any number of ledgers.
+Connectors take some risk, but this risk can be managed and is primarily based upon the connector's chosen ledgers and direct peers.
+
+## Cyclic transaction
+A transaction where the destination account is the same account, on the same ledger, as the source account.
+
+# Destination address
+The address of the receiver, included in the
+[interledger packet](#interledger-packet), and indicating
+to the ledger and account of the receiver.
+Note that anyone can claim to have a certain [Interledger address](#interledger-address), address ownership is not enforced.
+
+# Destination amount
+The amount to be received by the receiver.
+
+# Destination ledger
+The ledger pointed to by the destination address.
+
+## Exchange rate
+If the adjacent ledgers of a connector represent different assets, the connector sets the exchange rate between the transfers.
+Connectors may generate revenue from the difference in value between incoming and outgoing transfers.
+Senders may request quotes from multiple connectors to determine the best price before sending a payment.
+The exchange rate between source and destination is determined by the product of exchange rates at each hop.
+The application layer may provide guidance to sender and receiver in determining whether an exchange rate offered by a connector
+is reasonable or not.
+
+## Fulfillment
+See [conditional transfer](#conditional-transfer).
+
+## Destination (transfer, ledger, amount)
+The transfer/ledger/amount directly adjacent to the receiver.
+
+## Dispute
+Some ledgers, when administered by more than one authority, allow dispute over the ledger's state to occur naturally. This can be
+useful when the cost of resolving every small difference in accounting is much higher than the value of the differences.
+A transfer whose outcome is disputed among the ledger's administering authorities, can be resolved as 'disputed', in which case
+the sender's balance is decreased, and instead of increasing the receiver's balance, the sender's "disputed balance" is increased.
+Disputed balance can disappear over time when disputed transfers happen in the other direction, or when settled by out-of-band
+communication between the authorities who administer the ledger.
+Note that although individual ledgers can support disputed transfers, an interledger transaction as a whole can never be disputed;
+its final state can only be either 'executed' or 'canceled'.
+
+## Distributed ledger
+A ledger which is operated by a group of entities.
+
+## Executed
+One of the final states an interledger transactions can end up in.
+
+## Final state
+The state an interledger transaction ends up in. Has to be either
+[canceled](#canceled) or [executed](#executed).
+
+## Forward action
```

Comment:
Maybe it would be better to use the terms "Preparation Phase" and "Execution Phase", since that's how we normally describe this and it's more descriptive of what's going on than "forward" and "backward"